js_modules/dagit/packages/core/src/asset-graph/AssetNode.mocks.ts

```diff
@@ -543,6 +551,7 @@ export const LiveDataForNodePartitionedLatestRunFailed: LiveDataForNode = {
   freshnessPolicy: null,
   partitionStats: {
     numMaterialized: 1495,
+    numMaterializing: 0,
```

Comment:
If you use the `build` functions then when the schema changes like this you dont need to update any mocks!

Reply:
Ahh yeah, these ones all have specific values plugged in to test certain cases so I think they need to be made manually - going to try to use the build functions elsewhere though.